.github/workflows/pr-ci.yml

```diff
@@ -1,85 +1,153 @@
 name: CI build and push (PR)
 
 concurrency:
-  group: pr-${{ github.event.number }}
+  group: ci-${{ github.run_id }}
   cancel-in-progress: true
 
 on:
-  pull_request_target:
+  issue_comment:
     types:
-      - opened
-      - reopened
-      - synchronize
-      - labeled
-      - unlabeled
-    branches:
-      - main
-      - v[0-9]+
-      - v[0-9]+.[0-9]+
-      - cryostat-v[0-9]+.[0-9]+
+      - created
 
 jobs:
+  check-before-build:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'cryostatio' && github.event.issue.pull_request && startsWith(github.event.comment.body, '/build_test')
+    steps:
+    - name: Fail if needs-triage label applied 
+      if: ${{ contains(github.event.issue.labels.*.name, 'needs-triage') }}
+      run: exit 1
+    - name: Check command permission allowed
+      if: |
+        github.event.comment.author_association != 'MEMBER' &&
```

Comment:
This might be silly and depends on preferences. Do u think its better to write:

```bash
!(github.event.comment.author_association == 'MEMBER' ||
        github.event.comment.author_association == 'OWNER')
&& !contains(github.event.issue.labels.*.name, 'safe-to-test')
```

This translate to:

`If the user is neither member or owner and PR does not have the `safe-to-test` label.`